src/main/kotlin/calculator/domain/Delimiters.kt

```diff
@@ -0,0 +1,41 @@
+package calculator.domain
+
+import calculator.util.RegexUtil.customRegex
+
+class Delimiters(
+    private val value: List<String> = listOf(DEFAULT_DELIMITER_COMMA, DEFAULT_DELIMITER_COLONS),
+    private val operator: Operator = Operator.PLUS
+) {
+
+    fun toOperator(input: String): Operator {
+        return if (input in value) {
+            operator
+        } else {
+            throw IllegalArgumentException("등록된 구분자가 아닙니다.")
+        }
+    }
+
+    companion object {
+
+        private const val DEFAULT_DELIMITER_COMMA = ","
+        private const val DEFAULT_DELIMITER_COLONS = ":"
+
+        private const val INDEX_DELIMITER = 1
+
+        fun create(input: String): Delimiters {
+            val delimiter = customRegex.find(input)?.run {
+                groupValues[INDEX_DELIMITER]
+            }
```

Comment:
지금 같은 케이스는 `run` 보다는 `let` scope function을 쓰는 것이 더 적절한 것 같아요. 😃
```suggestion
            val delimiter = customRegex.find(input)?.let { it.groupValues[INDEX_DELIMITER] }
```

[코틀린 의 apply, with, let, also, run 은 언제 사용하는가?](https://medium.com/@limgyumin/%EC%BD%94%ED%8B%80%EB%A6%B0-%EC%9D%98-apply-with-let-also-run-%EC%9D%80-%EC%96%B8%EC%A0%9C-%EC%82%AC%EC%9A%A9%ED%95%98%EB%8A%94%EA%B0%80-4a517292df29) 글을 참고해보시면 좋을 것 같아요. 💪